examples/test_context_based_restrictions_v1_examples.py

```diff
@@ -0,0 +1,417 @@
+                account_id=TestContextBasedRestrictionsV1Examples.test_account_id
+            ).get_result()
+
+            print(json.dumps(rule_list, indent=2))
+
+            # end-list_rules
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_get_rule_example(self):
+        """
+        get_rule request example
+        """
+        try:
+            print('\nget_rule() result:')
+            # begin-get_rule
+
+            get_rule_response = context_based_restrictions_service.get_rule(
+                rule_id=TestContextBasedRestrictionsV1Examples.rule_id
+            )
+            rule = get_rule_response.get_result()
+
+            print(json.dumps(rule, indent=2))
+
+            # end-get_rule
+            TestContextBasedRestrictionsV1Examples.rule_rev = get_rule_response.headers.get("ETag")
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_replace_rule_example(self):
+        """
+        replace_rule request example
+        """
+        try:
+            print('\nreplace_rule() result:')
+            # begin-replace_rule
+
+            rule_context_attribute_model = {
+                'name': 'networkZoneId',
+                'value': TestContextBasedRestrictionsV1Examples.zone_id,
+            }
+
+            # Construct a dict representation of a RuleContext model
+            rule_context_model = {
+                'attributes': [rule_context_attribute_model],
+            }
+
+            # Construct a dict representation of a ResourceAttribute model
+            account_id_resource_attribute_model = {
+                'name': 'accountId',
+                'value': TestContextBasedRestrictionsV1Examples.test_account_id,
+            }
+
+            service_name_resource_attribute_model = {
+                'name': 'serviceName',
+                'value': TestContextBasedRestrictionsV1Examples.service_name,
+            }
+
+            # Construct a dict representation of a ResourceTagAttribute model
+            resource_tag_attribute_model = {
+                'name': 'tagName',
+                'value': 'tagValue',
+            }
+
+            # Construct a dict representation of a Resource model
+            resource_model = {
+                'attributes': [account_id_resource_attribute_model, service_name_resource_attribute_model],
+                'tags': [resource_tag_attribute_model],
+            }
+
+            rule = context_based_restrictions_service.replace_rule(
+                rule_id=TestContextBasedRestrictionsV1Examples.rule_id,
+                if_match=TestContextBasedRestrictionsV1Examples.rule_rev,
+                contexts=[rule_context_model],
+                resources=[resource_model],
+                description='SDK TEST - this is an example of updated rule'
+            ).get_result()
+
+            print(json.dumps(rule, indent=2))
+
+            # end-replace_rule
```

Comment:
I'd like to ask you to remove the comments between `# end-replace_rule` and `# begin-replace_rule`.